src/main/java/de/kekru/struktogrammeditor/control/Struktogramm.java

```diff
@@ -1,4 +1,5 @@
 package de.kekru.struktogrammeditor.control;
+

```

Comment:
What happened in this file? It is difficult to see the differences with the new formatting

Reply:
Maybe just newline?